tests/functional/adapter/common.py

```diff
@@ -0,0 +1,136 @@
+from typing import Dict, Tuple, List
+
+from dbt.tests.util import relation_from_name
+from dbt.tests.fixtures.project import TestProjInfo
+
+
+def get_records(project: TestProjInfo, table: str, select: str = None, where: str = None) -> List[Tuple]:
```

Comment:
looks like these aren't redshift specific, should we just open a PR in core to add them?

Reply:
Eventually. My approach is:
1) put them in dbt-redshift
2) get feedback on whether we want to make changes
3) open a PR to put them in dbt-core
4) open a PR to removed them from dbt-redshift and use the dbt-core versions

That way we're not waiting on PRs from two different repos to get this out or point to two different branches of dbt-core like we do in other tests.